src/routes/nearYouPage/NearYouPage.js

```diff
@@ -0,0 +1,20 @@
+import React, {useState} from 'react';
+import {Topbar} from '../../components';
+import styled from 'styled-components';
+
+const StyledText = styled.h3`
+    margin-top: 20px;
+    text-align: center;
+`
+
+const NearYouPage = () => {
+    const [noResults, setNoResults] = useState(false);
+    return (
+        <div>
+            <Topbar title="Study spaces near you" hasBack={true} titleSize={'24pt'}/>
```

Comment:
same for this page. like maybe a "Galbraith Hall infinity miles away" or whatever. personally feel like showcasing UI elements is important for this assignment because it'll be a bit of a headache to do extra UI elements PLUS functionality next week